sdk/storage/azqueue/models.go

```diff
@@ -80,6 +80,9 @@ type Queue = generated.Queue
 // AccessPolicy - An Access policy
 type AccessPolicy = generated.AccessPolicy
 
+// GeoReplicationStatus - The status of the secondary location
+type GeoReplicationStatus = generated.GeoReplicationStatus
```

Comment:
The pattern we've been following is to place all of the `const` types and their respective values in a `constants.go` file.

Reply:
that makes sense, will make that change